framework/src/modules/chain/processor/processor.js

```diff
@@ -158,9 +158,8 @@ class Processor {
 		});
 	}
 
-	async forkStatus(receivedBlock) {
+	async forkStatus(receivedBlock, lastBlock = this.blocksModule.lastBlock) {
```

Comment:
I don't like this to have this static getter being used as default value directly in the parameter definition (personal opinion).

Reply:
Sometimes we want to pass an optional lastBlock to statically compare two blocks (BSM)

Reply:
I agree on the use, but I don't like to have like business logic tied to our input parameters, feels bit odd?
this default value is more meant to be like object = {} or string = '' but`this.blocksModule.lastBlock` is really business logic and the function depends on it

Reply:
It is meant to set default values, and in this case, the last block has default value the real last block of the chain. I get your point but I don't think it is misleading.

Reply:
I also agree with @michielmulders 
Default input should be static to be able to test easier.
It could be optional and assigned if not exist?